tests/logger/test_handlers.py

```diff
@@ -49,10 +49,20 @@ def socket_mocks(mocker):
 
 
 @pytest.fixture()
-def broken_pipe_error(mocker):
-    mock_exc_info = mocker.patch("code42cli.logger.handlers.sys.exc_info")
-    mock_exc_info.return_value = (BrokenPipeError, None, None)
-    return mock_exc_info
+def system_exception_info(mocker):
+    return mocker.patch("code42cli.logger.handlers.sys.exc_info")
+
+
+@pytest.fixture()
+def broken_pipe_error(system_exception_info):
+    system_exception_info.return_value = (BrokenPipeError, None, None)
+    return system_exception_info
+
+
+@pytest.fixture()
+def connection_reset_error(system_exception_info):
+    system_exception_info.return_value = (ConnectionResetError, None, None)
```

Comment:
This is the error I saw during Windows, so the test explicitly checks for that rather